src/lib/phdi-building-blocks/tests/test_linkage.py

```diff
@@ -109,7 +112,7 @@ def test_add_patient_identifier():
         "use": "temp",
     }
```

Comment:
I think our unit tests will be more readable if we move this "testing data" into JSON files.